src/components/ChannelList/hooks/useNotificationRemovedFromChannelListener.js

```diff
@@ -0,0 +1,29 @@
+import { useEffect, useContext } from 'react';
+import { ChatContext } from '../../../context';
+
+export const useNotificationRemovedFromChannelListener = (
+  setChannels,
+  customHandler,
```

Comment:
Without listing this `customHandler` as a dependency on the `useEffect` hook, we make it no longer possible to use dynamic custom handlers. Looking at [the old implementation](https://github.com/GetStream/stream-chat-react/pull/236/files#diff-23769ba2a0759e37502ac29ef02b9795L298) it seems like this was possible before.

Reply:
The same is true for other listeners in this PR.